test/index.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+require('assert');
+const should = require('should');
+
+describe('$', function () {
+  before(function () {
+  });
+
+  after(function () {
+  });
+
+  it('should export properly', function () {
+    should.exist(require('..'));
```

Comment:
I'm not sure if using something besides `.exist()` would be better here. Seems like this is a good enough check to me though.